.travis.yml

```diff
@@ -1,3 +1,15 @@
-
+language: Python
+Python:
+  - "3.7"
+before_install:
+  # require for installing pyodbc
+  - sudo apt-get install unixodbc-dev
+# command to setup datamegh
+install:
+  - make setup
+# check for black format
+# build will stop if check fail 
+before_script:
+  - black --check --diff . 
```

Comment:
I would also suggest adding this as a make command so that while in your local you can also do something like this:
```bash
$ make black_check
```